### PR TITLE
task 9

### PR DESCRIPTION
**Pull request by NayakRoshan**
task9_merged

### CODE REVIEW

File: app/src/main/java/com/example/mytaskapp/DetailsListView.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.example.mytaskapp.entity.PersonEntity;
+
+import java.util.List;
+
+public class DetailsListView extends RecyclerView.Adapter<DetailsListView.DetailsViewHolder> {
```
Comment (rishabh-jain-1990) [inline app/src/main/java/com/example/mytaskapp/DetailsListView.java]: @NayakRoshan class name should indicate the parent class. `DetailsListView` signifies its a ListView implementation while in reality it's a `RecyclerView.Adapter`

File: app/src/main/res/layout/list_layout.xml [hunk 2]
```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <LinearLayout
```
Comment (rishabh-jain-1990) [inline app/src/main/res/layout/list_layout.xml]: @NayakRoshan no nesting

File: app/src/main/res/layout/list_layout.xml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        android:orientation="vertical"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent">
+
+        <TextView
+            android:id="@+id/id"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
```
Comment (rishabh-jain-1990) [inline app/src/main/res/layout/list_layout.xml]: @NayakRoshan color hardcoding

**Review by rishabh-jain-1990 — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/mytaskapp/DetailsListView.java (hunk 1 above)
Comment (NayakRoshan) [reply app/src/main/java/com/example/mytaskapp/DetailsListView.java]: Changed the class name to DetailsRecyclerViewAdapter

File: app/src/main/res/layout/list_layout.xml (hunk 3 above)
Comment (NayakRoshan) [reply app/src/main/res/layout/list_layout.xml]: color added to color xml file

**Review by rishabh-jain-1990 — APPROVED** [review]

**Pull request merged**